services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -129,7 +136,8 @@ class ProjectSummary(SoftDeleteMixin, AuditMixin, Base):
     )
 
     municipality = db.relationship(
-        'Municipality', lazy='joined', foreign_keys=nearest_municipality_guid
+        'Municipality', lazy='joined', foreign_keys=nearest_municipality_guid,
```

Comment:
Ah, will this address that municipality warning/message that pops up in the logs?

Reply:
Yes!